eZ/Bundle/EzPublishCoreBundle/Tests/ApiLoader/RepositoryConfigurationProviderTest.php

```diff
@@ -93,6 +87,48 @@ public function testGetRepositoryConfigUndefinedRepository()
         $provider->getRepositoryConfig();
     }
 
+    /**
+     * @dataProvider providerForRepositories
+     */
+    public function testGetDefaultRepositoryAlias(array $repositories): void
+    {
+        $configResolver = $this->getConfigResolverMock();
+
+        $provider = new RepositoryConfigurationProvider($configResolver, $repositories);
+        $provider->getRepositoryConfig();
+
+        $this->assertSame('first', $provider->getDefaultRepositoryAlias());
```

Comment:
EA: use self for static methods:
```suggestion
        self::assertSame('first', $provider->getDefaultRepositoryAlias());
```